cmd/main.go

```diff
@@ -73,6 +73,7 @@ var (
 	startupMemoryProfile = flag.String("startup_memprof", "", "File path to write the memory profile of mixer startup to")
 	// Serve live profiles of the process (CPU, memory, etc.) over HTTP on this port
 	httpProfilePort = flag.Int("httpprof_port", 0, "Port to serve HTTP profiles from")
+	pinnedSvg       = flag.String("pinned_svg", "", "When exist, this will be a top level svg and other other top level svg will be grouped together")
```

Comment:
nit: "When set, this will be the ..."
nit: typo "other other"

Reply:
Done